server/bin/pbench-audit-server.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+import sys
+import glob
+import re
+import errno
+import hashlib
+import shutil
+import tempfile
+import tarfile
+from enum import Enum
+from argparse import ArgumentParser
+from s3backup import S3Config, Entry
+
+from pbench import PbenchConfig, BadConfig, get_pbench_logger, md5sum
+from pbench.report import Report
+
+
+_NAME_ = "pbench-audit-server"
+
+def verify_subdirs(config, controller, cnt):
+
+    linkdirs = sorted(config.LINKDIRS.split(' '))
+
+    states = glob.iglob(os.path.join(controller,'*'))
+    tarball, symlink, unexpected_objects, unexpected_dirs = ([] for i in range(4))
+
+    # Checking if the list is empty or not
+    if len(os.listdir(controller)) == 0:
+        print("\t* No state directories found in this controller directory.")
+    else:
+        for state in states:
+            st = os.path.basename(os.path.normpath(state))
+            if st != '_QUARANTINED' and ('WONT-INDEX' not in st):
```

Comment:
Can we use `st.startswith('WONT-INDEX')` instead?  This makes it a bit more explicit for how the directory name is formed.

Reply:
Also, for `_QUARANTINED` we also need to check that it is a directory, and for the `WONT-INDEX` names as well.

So can we make this a positive test, and then use a `continue` statement?  This would avoid another level of indentation for the code following.